cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/repo.go

```diff
@@ -250,6 +327,141 @@ func indexOneRepo(unstructuredRepo map[string]interface{}) (charts []models.Char
 	return charts, nil
 }
 
+// onModifyRepo essentially tells the cache whether to and what to store for a given key
+func (s *repoEventSink) onModifyRepo(key string, unstructuredRepo map[string]interface{}, oldValue interface{}) (interface{}, bool, error) {
```

Comment:
We had a way to manually force a resync in the AppRepo (via asset-syncer), I'd say it was by adding a `resync: <new value>` in the k8s CR.
I guess it still will be detected as a change by this eventSink, isn't it?

Reply:
0) I am not removing any existing functionality from kubeapps AppRepo
1) let me point out that in general, flux is not kubeapps AppRepo, so it's not likely have a set of the same features. Taking a cursory look at their API I don't see a way to force a manual resync.
2) if you are referring to 	ResyncRequests     uint                   `json:"resyncRequests"`
field in type AppRepositorySpec, we (me, Michael and Dimitri) actually had a chat about this field when discussing new RepositoriesService API. You were on vacation, I think. All present agreed it was a hack and did not merit a place in the new API. https://docs.google.com/document/d/1z03msZRGsRvcRaom-yrvEwIWcEfNy6fSA5Zg28gjYvA/edit#heading=h.yheooaol1kzd, page 28, AddPackageRepository section
3) having said that, if there is a genuine need to have a manual resync feature, we need to design it such that it works for all plugins. So far I haven't heard the need, but will be on the look out

Reply:
Thanks for the clarification. Yep, I was referring to `ResyncRequests` field; I wasn't aware of the outcome of the meeting . I do agree it's was a hacky way to force a resync.
ATM, I don't have any strong opinion, though I feel the "sync" button, from a user's point of view, is important. Perhaps we could trigger a kind of "kubectl apply" command to still trigger an update event somehow.

Nevertheless, this discussion does not belong in this PR, but the AppRepo API refactor issue. Thanks for the context again!